packages/combo-box/test/overlay-position.test.js

```diff
@@ -50,11 +50,6 @@ describe('overlay position', () => {
     await aTimeout(0);
   });
 
-  // Clean out <vaadin-overlay> elements from body.
-  afterEach(() => {
-    comboBox.close();
-  });
```

Comment:
This isn't needed since we have `disconnectedCallback` in `ComboBoxMixin` that handles overlay cleanup.